openid-federation-1_0.xml

```diff
@@ -3739,7 +3741,7 @@
             <artwork><![CDATA[
```

Comment:
The kid in the JWS header parameters of the delegation claim is wrong. 
Given as "TjNZeEdqdDRPSXpoUlhLOUZjcXNJb2xjci02VGxCMWRkVUUxMGhjNjd0MA" but
should be "kosGm2wuZh9DGmNxAtkuO40pPjpL3-jCf1N-qPO-YeU".

Reply:
Change made in https://github.com/openid/federation/commit/e83739c8bdcaf12fa41c06bf39bd1880051fef80 , which completed the `id` -> `trust_mark_id` changes.